sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -58,14 +65,17 @@
 
 import static com.azure.core.amqp.AmqpMessageConstant.PARTITION_KEY_ANNOTATION_NAME;
 import static com.azure.core.amqp.AmqpMessageConstant.SCHEDULED_ENQUEUE_UTC_TIME_NAME;
+import static com.azure.messaging.servicebus.implementation.MessageUtils.describedToDuration;
+import static com.azure.messaging.servicebus.implementation.MessageUtils.describedToOffsetDateTime;
+import static com.azure.messaging.servicebus.implementation.MessageUtils.describedToURI;
```

Comment:
Avoid static import, especially for method.

Reply:
Sure, I'll fix this.